minigotchi-ESP32/display.h

```diff
@@ -1,29 +1,33 @@
-/**
- * display.h: header files for display.cpp
- */
-
 #ifndef DISPLAY_H
 #define DISPLAY_H
 
-#include "config.h"
-#include <Adafruit_GFX.h>
 #include <Adafruit_SSD1306.h>
-#include <SPI.h>
 #include <Wire.h>
+#include <TFT_eSPI.h>  // Defines the TFT_eSPI library for CYD
+#include "Config.h"
 
 // SSD1306 screen
 #define SSD1306_SCREEN_WIDTH 128
-#define SSD1306_SCREEN_HEIGHT 64 
+#define SSD1306_SCREEN_HEIGHT 64
 
 // Init screen
 #define SSD1306_OLED_RESET -1 
 #define WEMOS_OLED_SHIELD_OLED_RESET 0 // GPIO0
 
+// Define display parameters for CYD
+#define CYD_SCREEN_WIDTH 240
+#define CYD_SCREEN_HEIGHT 320
+
 class Display {
 public:
   static void startScreen();
   static void cleanDisplayFace(String text);
   static void attachSmallText(String text);
+  ~Display();
```

Comment:
im not 100% sure on this, a deconstructor that deletes pointers can save memory, but it may lead to an exception if the pointer to the screen is removed, something along those lines has happened while I was testing.